jest.config.js

```diff
@@ -0,0 +1,14 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+const { pathsToModuleNameMapper } = require('ts-jest');
+const { compilerOptions } = require('./tsconfig');
+
+module.exports = {
+  preset: 'react-native',
+  setupFiles: ['./config/test/jest-setup.js'],
+  transform: {
+    '\\.tsx?$': 'ts-jest',
```

Comment:
Slight variation from the e2e config to allow other files to fall back to default Jest handling that's configured in the `react-native` preset. In Detox we can't set this preset (doesn't work), so we defer everything to `ts-jest`. Appears to work just fine.